llvm/include/llvm/IR/User.h

```diff
@@ -54,21 +54,24 @@ class User : public Value {
   void *operator new(size_t Size) = delete;
 
   /// Indicates this User has operands "hung off" in another allocation.
-  struct HungOffOperandsAllocMarker {};
+  struct HungOffOperandsAllocMarker {
+    /// The number of operands for this User.
+    unsigned NumOps;
+  };
 
   /// Indicates this User has operands co-allocated.
   struct IntrusiveOperandsAllocMarker {
     /// The number of operands for this User.
-    const unsigned NumOps;
```

Comment:
Why un-const these?

Reply:
This is for consistency, because in `HungOffOperandsAllocMarker` the members are also non-const.